src/firebase/dynamicLinks.ts

```diff
@@ -26,8 +26,22 @@ export async function createInviteLink(address: string) {
 }
 
 export async function createJumpstartLink(privateKey: string, networkId: NetworkId) {
-  return dynamicLinks().buildLink({
-    ...commonDynamicLinkParams,
-    link: `${WEB_LINK}jumpstart/${privateKey}/${networkId}`,
-  })
+  // avoid calling firebase sdk with private key during link creation to protect
+  // the private key from being stored
+  const dynamicLink = new URL(
+    await dynamicLinks().buildLink({
+      ...commonDynamicLinkParams,
+      link: WEB_LINK,
+    })
+  )
+  const dynamicUrl = new URL(dynamicLink)
```

Comment:
maybe i'm misunderstanding something here, but do we need to wrap this with another `new URL` constructor? isn't this essentially `new URL(new URL(stuff...))`?

Reply:
oops, i meant to remove that. thanks for the catch!